server/filestore.go

```diff
@@ -2614,10 +2614,6 @@ func (fs *fileStore) numFilteredPendingWithLast(filter string, last bool, ss *Si
 	// Always reset.
 	ss.First, ss.Last, ss.Msgs = 0, 0, 0
 
-	if filter == _EMPTY_ {
```

Comment:
This is redundant, if `isAll := filter == _EMPTY_ || filter == fwcs` we early return above.